c_emulator/riscv_sim.c

```diff
@@ -245,8 +250,10 @@ char *process_args(int argc, char **argv)
 #ifdef SAILCOV
                     "c:"
 #endif
-                         , options, NULL);
-    if (c == -1) break;
+                    ,
```

Comment:
I have no idea why this is happening and can't find an easy way to work around it.... Probably due to the ifdef before